docs/book/content/advanced/subscriptions.md

```diff
@@ -0,0 +1,165 @@
+# Subscriptions
+### How to achieve realtime data with GraphQL subscriptions
+
+GraphQL Subscriptions are a way to push data from the server to the clients that choose to listen to real time messages 
+from the server. Subscriptions are similar to queries in that they specify a set of fields to be delivered to the client,
+but instead of immediately returning a single answer, a result is sent every time a particular event happens on the 
+server. 
+
+For the usage of subscriptions you need to add [juniper_subscriptions][juniper_subscriptions] into your cargo.toml:
```

Comment:
I'd add here a brief explanation about subscriptions coordinators and why we need to add juniper_subscriptions into a cargo.toml:

```
In order to execute subscriptions you need a coordinator (that spawns connections) 
and a GraphQL object that can be resolved into a stream, elements of which will then 
be returned to the end user. [juniper_subscriptions][juniper_subscriptions] crate 
provides a default connection implementation, and you'll need to add it into 
your cargo.toml:
```